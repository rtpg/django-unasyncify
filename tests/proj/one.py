# ruff: noqa


@generate_unasynced
async def aget_thing(qs):
    return await qs.aget()


@generate_unasynced
async def ado_thing():
    if IS_ASYNC:
        return 1
    else:
        return 2
