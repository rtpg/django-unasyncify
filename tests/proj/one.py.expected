# ruff: noqa


from unasync_utils import IS_ASYNC, from_codegen, generate_unasynced

@from_codegen
def get_thing(qs):
    return qs.get()

@generate_unasynced
async def aget_thing(qs):
    return await qs.aget()


@from_codegen
def do_thing():
    return 2



@generate_unasynced
async def ado_thing():
    if IS_ASYNC:
        return 1
    else:
        return 2
